packages/_site/src/pages/components/routes_list_docs_components.ts

```diff
@@ -188,7 +188,7 @@ export const componentsList: I_ComponentsList[] = [
 
   {
     title: "Switch",
-    description: "",
+    description: "A switch is a control used to toggle between enabled or disabled states.",
```

Comment:
Let's change this to true and false state, since we use a disabled prop to fully disabled interactive off the component, for better clarity.